tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -39,7 +39,7 @@ use tracing_core::{
 #[derive(Debug)]
 pub struct EnvFilter {
     // TODO: eventually, this should be exposed by the registry.
-    scope: thread::Local<Vec<LevelFilter>>,
+    scope: thread::Local<Vec<(Option<Arc<str>>, LevelFilter)>>,
```

Comment:
TIOLI: What do you think about changing this tuple into a struct of some kind? The code might be clearer if we had field names...

Reply:
Yeah, I wanted to do that but couldn't find a good name...

Reply:
If the target becomes a newtype, it is probably fine for this to stay a tuple.